src/jit/codegenxarch.cpp

```diff
@@ -2645,8 +2576,6 @@ CodeGen::genCodeForTreeNode(GenTreePtr treeNode)
 bool
 CodeGen::genStoreRegisterReturnInLclVar(GenTreePtr treeNode)
```

Comment:
A suggestion to reduce #ifdef clutter around the code where this method is used:

Let this routine be defined in all targets. It will return false for x86 and amd64 windows.

Enclose the entire if stmnt in this routine (starting at line 2579) under #ifdef FEATURE_UNIX_AMD64_STRUCT_PASSING